frontend/src/app/common/validation-checkbox-group.tsx

```diff
@@ -32,10 +32,6 @@ export const ValidationCheckboxGroup: FC<ValidationCheckboxGroupProps> = ({
     onCheckboxChange(updatedCheckedItems);
   };
 
-  useEffect(() => {
```

Comment:
The component is already initializing to checkedValues and handling changes internally by calling setCheckedItems in handleCheckboxChange. This effect doesn't work as intended and just removing it does not break any current usage of this component.